src/PresentationalComponents/SimpleTableFilter/styles.scss

```diff
@@ -1,2 +1,3 @@
+@import '~@patternfly/patternfly-next/sass-utilities/all';
 @import '~@patternfly/patternfly-next/components/InputGroup/styles.scss';
-@import '~@patternfly/patternfly-next/components/FormControls/styles.scss';
+@import '~@patternfly/patternfly-next/components/FormControl/styles.scss';
```

Comment:
This should be
`@import '~@patternfly/patternfly-next/components/FormControls/styles.scss';`

Reply:
^^ was updated to `FormControl/`